eventonica-api/models/user.js

```diff
@@ -0,0 +1,31 @@
+const { Sequelize, DataTypes, Model } = require('sequelize');
+// const sequelize = new Sequelize('sqlite::memory:');
```

Comment:
As another style nit, don't checked in commented out code. Just delete it. If you need to see what you were doing before, look in the git history not code comments.